### ISSUE TITLE
Select project from different organization for issues

### ISSUE THREAD

**Issue by serkan-ozturk** [go-gitea/gitea#24088]
### Feature Description

I would like to be able to assign issue to project from different organization.
Currently, gitea allows me to only select project from current repository's organization.

**Comment by lunny** [comment]
So could you move those repositories into the same organization? Cross organization projects maybe too much.

**Comment by serkan-ozturk** [comment]
Yes I know. Moving repositories under one organization is a solution but sometimes we need organizational isolation in order to seperate some repositories. And with this, we need to assign issue to project from different organization.
It would be nice for our workflow if we could able to to this.
